client/question/components/LearnerViewDirective.js

```diff
@@ -983,6 +985,15 @@ tie.directive('learnerView', [function() {
         };
 
         $scope.initQuestionSet(questionSetId);
+
+        // If server version, and the user has not accepted the privacy policy,
+        // show them the privacy modal.
+        if (SERVER_URL) {
+          var privacyPolicyAccepted = CookieStorageService.getPrivacyCookie();
```

Comment:
Do we need to check expiry, or is this manifested by the disappearance of the cookie? (You can test using a cookie of short duration to see what happens, but it might be prudent to just check non-expiry anyway.)

Reply:
Well I think the cookie should expire and return undefined (which getPrivacyCookie will return as false), and I can test on the server soon. (Locally cookies don't work).